wolfcrypt/src/random.c

```diff
@@ -2204,30 +2204,49 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         return 0;
     }
 #elif defined(WOLFSSL_VXWORKS)
+    #ifdef WOLFSSL_VXWORKS_6_x
+        #include "stdlib.h"
+        #warning "potential for not enough entropy, currently being used for testing"
+        int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
+        {
+            int i;
+            int seed = XTIME(0);
```

Comment:
XTIME returns `time_t`, which isn't always `int`. Consider explicit cast on lines where seed is set.

Reply:
@dgarske, good catch. fixed.